libs/inspecjs/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "inspecjs",
+  "version": "2.4.6",
+  "description": "© 2018 The MITRE Corporation.",
+  "files": [
+    "dist"
+  ],
+  "main": "./src/inspecjs.ts",
+  "dependencies": {},
+  "devDependencies": {
+    "@types/node": "^12.20.7",
+    "@typescript-eslint/eslint-plugin": "^2.34.0",
+    "@typescript-eslint/parser": "^2.34.0",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.15.0",
+    "eslint-plugin-prettier": "^3.3.1",
+    "prettier": "^1.19.1",
+    "typedoc": "^0.20.35",
+    "typescript": "^3.9.9"
+  },
+  "scripts": {
+    "build": "rm -r ./dist; tsc",
+    "docs": "typedoc --out docs ./src",
+    "gen-types": "bash scripts/update_schema.sh",
+    "lint": "eslint --ext .ts src; exit 0"
```

Comment:
Not sure what the point of eslint is here if we are just going to make it always pass. Can this be updated to look how our other linters look?